specification/ml/_types/TrainedModel.ts

```diff
@@ -157,3 +157,11 @@ export enum TrainedModelType {
    */
   pytorch
 }
+
+export class TrainedModelEntities {
+  class_name: string
+  class_probability: double
+  entity: string
+  start_pos: number
```

Comment:
Use `integer`/`long`/`float`/`double` instead of `number` so strongly typed languages don't have to guess. Might be something to add to the best practices document.

Reply:
Thanks! I've changed it to integers in aec74e54f04c763e7e37c72c109098c54fd5fe7b